charts/astronomer/values.yaml

```diff
@@ -2,6 +2,20 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+global:
```

Comment:
by adding the used global variables down in each subchart, this allows the charts to be valid independently, which is important for linting.